resources/charts/binary-runner/templates/pod.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "binary-runner.fullname" . }}
+  labels:
+    {{- include "binary-runner.labels" . | nindent 4 }}
+    app: {{ include "binary-runner.name" . }}
+    mission: binary
+spec:
+  restartPolicy: Never
+  volumes:
+    - name: shared-data
+      emptyDir: {}
+  initContainers:
+    - name: {{ .Values.pod.name }}-receiver
+      image: alpine
+      command: ["/bin/sh", "-c"]
+      args:
+        - |
+          echo "Waiting for binary file..."
+          while [ ! -f /data/binary ]; do
```

Comment:
If this wait was moved into the main container we wouldn't need the initContainer?

Is there another advantage to the init container?

Reply:
There is no other advantage, nice spot. This is just an artefact of me trying to migrate from `configMap` and, instead of taking two steps forward, only taking one.

Removed the init container in [08b15ed](https://github.com/bitcoin-dev-project/warnet/pull/595/commits/08b15eda8705a65ca02351ddf50c065adb90b746)